<!DOCTYPE HTML>
<!--
	Apex by Pixelarity
	pixelarity.com | hello@pixelarity.com
	License: pixelarity.com/license
-->
<html>
	<head>
		<title>UW Genome Sciences: Hackathon</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="../assets/css/main.css" />
		<style type="text/css">
		.style30 {font-family: Arial, Helvetica, sans-serif}
.style34 {font-weight: bold}
.style36 {font-size: 10pt; font-weight: bold; }
.style37 {font-family: Arial, Helvetica, sans-serif; font-size: 10pt; font-weight: bold; }
.style44 {font-size: 10pt}
        </style>
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
	</head>
	<body>
		<div id="page-wrapper">

			<!-- Header Wrapper -->
				<div id="header-wrapper">

					<!-- Header -->
						<div class="container">
							<header id="page-header">
								<h1><a href="../index.htm" id="logo"><span>UW Genome Sciences</span></a></h1>
								<nav id="nav">
									<ul>
										
                                        <li><a href="../index.htm">Home</a></li>
                                        <li>
											<a href="../about/index.htm" class="arrow">About</a>
											<ul>
												<li><a href="../about/welcome.htm">Welcome</a></li>
												<li><a href="../about/contact.htm">Contact</a></li>
												<li><a href="../about/directory/index.htm">Directory</a></li>
                                                <li><a href="../about/history.htm">History</a></li>
                                                <li><a href="../about/support.htm">Support Genome Sciences</a></li>
											</ul>
										</li>
                                        <li><a href="../faculty/index.htm">Faculty</a></li>
                                        <li>
											<a href="../academics/index.htm" class="arrow">Academics</a>
											<ul>
												<li><a href="../academics/gradprogram/index.htm">Ph.D. Program</a></li>
												<li><a href="../office/postdocs/index.htm">Postdoctoral Program</a></li>
												<li><a href="../academics/courses/index.htm">Courses</a></li>
												<li><a href="../academics/tg.htm">Training Grants</a></li>
                                                <li><a href="../academics/summer/index.htm">Summer Research</a></li>
                                                <li><a href="../academics/diversity.htm">Diversity</a></li>
                                                
											</ul>
										</li>
										<li>
											<a href="index.htm" class="arrow">News & Events</a>
											<ul>
												<li><a href="calendar.htm">Events Calendar</a></li>
												<li><a href="seminar.htm">GS Seminar</a></li>
												<li><a href="combi.htm">Combi Seminar</a></li>
                                                <li><a href="journal.htm">Journal Club</a></li>
												<li><a href="reports.htm">Research Reports</a></li>
                                                <li><a href="outreach.htm">Outreach</a></li>
                                                <li><a href="wigs.htm">Women In Genome Sciences (WiGS)</a></li>
                                                <li><a href="salon.htm">Genomics Salon</a></li>
                                                <li><a href="news.htm">Department News</a></li>
                                                <li><a href="itcalendar.htm">IT Outages</a></li>
                                                
											</ul>
										</li>
										
										<li>
											<a href="#" class="arrow">Administration</a>
											<ul>
												<li><a href="../computing/index.htm">Computing</a></li>
                                                <li><a href="../office/rooms.htm">Conference Rooms</a></li>
												<li><a href="../about/directory.htm">Directory</a></li>
												<li><a href="../office/employment/index.htm">Employment</a></li>
                                                <li><a href="../office/facilities/index.htm">Facilities</a></li>
                                                <li><a href="../office/facultyresources/index.htm">Faculty Resources</a></li>
                                                <li><a href="../office/operations/hr/index.htm">Human Resources</a></li>
                                                <li><a href="../office/operations/index.htm">Operations</a></li>
                                                <li><a href="../office/researchsupport/index.htm">Research Support</a></li>
                                              
                                                
											</ul>
										</li>
									</ul>
								</nav>
							</header>
						</div>

				</div>

			 <!-- Search Wrapper -->
				<div id="search-wrapper">

					<!-- Search -->
						<div class="container" id="search">
							<span>Model Organism Genetics -- Human and Medical Genetics -- Genomics and Proteomics -- Computational Biology </span>
							<div class="row">
								<div class="-8u 12u(mobile) 4u 12u(mobile)">
									<div class="side-padded">
										
                                        <form method="GET" action="http://www.google.com/search">
											<div>
												<input type=hidden name=ie value=UTF-8>
              <input type=hidden name=oe value=UTF-8>
              <INPUT TYPE=text name=q size=40 maxlength=255 value="">
                       
                      <input type=hidden name=sitesearch value="http://www.gs.washington.edu">        
											</div>
										</form>
                                        
									</div>
								</div>
							</div>
						</div>

				</div>

			<!-- Main -->
				<div id="main-wrapper">
					<div class="container">
						<div class="row">
							<div class="12u important(mobile)">

								<!-- Content -->
									<div id="content">

										<!-- Article -->
											<article class="featured">
												<header>
													<h2>Genome Sciences Hackathon</h2>
													<p>&nbsp;</p>
												</header>
												
											   <div> 
                                                 <p align="center"><img src="images/hackathon.png" width="355" height="178" alt=""/ ></p>
                                                 <p>Do you love innovation and challenge? Do you  like problem solving in collaborative environments? Do you want to learn new  technical skills outside of your current projects? Most importantly, do you  want to make tools that will improve the field of genomics and make it more  accessible to the public?</p>
                                                 <p>If so, the department of Genome Sciences  proudly invites you to participate in our second annual Hackathon! The  Hackathon will span 5 days, from Monday September 18th through Friday September  22nd. All GS trainees and faculty (graduate, post-doctoral, and staff) are  invited to participate. There will be four teams led by your amazing  colleagues, where each team will build a deliverable by the end of the five  days. This is an opportunity to build something cool, learn from colleagues  outside of your lab, and have fun!</p>
                                                 <p>If you&rsquo;re interested, please read the project  details below and fill out the interest form: <br>
                                                   <a href="https://forms.gle/eDxhcwNoa34wwkSm8" target="_blank" class="aaastandout">https://forms.gle/eDxhcwNoa34wwkSm8</a></p>
                                                 <p>If you have any questions about the event,  please contact Sayeh Gorjifard &lt;sgorji [ a t ] uw.edu&gt;</p>
                                                 <p class="aaastandout"><strong>EVENT DETAILS:</strong><br>
                                                 <strong>Location: Each team will have a  conference room in Foege  <br>
                                                   Dates: September 18-22, 2023<br>
                                                   Times: 9 am - 5 pm (with afternoon review sessions in Foege auditorium 3 pm on  Monday and Wednesday and a final presentation on Friday). </strong><br>
                                                 <strong>Food:  Coffee/snacks are provided daily<br>
    Final celebration (beer hour + food) will be Friday at 5 pm on the 3rd floor  common area.</strong></p>
                                                 <p><span class="aaastandout"><strong>Project #1: </strong><br>
                                                 <strong>Leading  lab</strong>:</span> Dunham Lab<br>
                                                   <span class="aaastandout"><strong>Stakeholders</strong>:</span>  Maitreya Dunham and Leah  Anderson <br>
                                                   <strong class="aaastandout">Hackathon  project title: </strong>Developing bioinformatics and data  visualization resources for yEvo (<a href="https://yevo.org/" target="_blank" class="aaastandout">https://yevo.org/</a>), which teaches high  schoolers about evolution and genetics using yeast.<br>
                                                   <span class="aaastandout"><strong>Desired  deliverable</strong>:</span> Website tools for high school students to  see and interact with whole genome sequencing data, along with curricular  materials to guide students through a lab about whole genome sequencing and  bioinformatics. Long term deliverable is to offer a bioinformatics module as a  standalone product that teachers could use in their classes without direct  intervention from us. If this is successful,  other yeast labs that have less access to computational resources would  probably use it, and we hope that SGD will host the tool.  <br>
                                                 <span class="aaastandout">Expected coding experience level: </span>Beginner-Intermediate-Advanced</p>
                                                 <p><span class="aaastandout">Abstract: </span><br>
                                                 Whole-genome sequencing is not accessible for  users who lack access to expert training or coding expertise, like biologists  at non-elite schools or high schoolers. To address this problem, the desired  deliverable will be a web-based tool that  associated protocol users can use to analyze and visualize their whole genome  sequencing data. We aim to have a simple user interface that would allow the  students to understand each step in the analysis pipeline. We were  thrilled that yEvo was one of the Hackathon teams last year, as we were able to  move our sequencing data analysis pipeline into Snakemake and build a Shiny  app, Mutation Browser: <a href="https://yevo.org/mutation-browser/" target="_blank" class="aaastandout">https://yevo.org/mutation-browser/</a>. To expand  the tool further, we aim to create a website where students can see pile-ups  and mutation calls and interact with their &ldquo;raw&rdquo; data. Further improvements to  the Mutation Browser Shiny app will allow students to compare mutations across  and between classes, and to link out to SGD and other resources to learn about  their mutations. The stretch goal would be to build  this as a website to help all yeast geneticists. </p>
                                                 <p><strong>&nbsp;</strong></p>
                                                 <p><span class="aaastandout"><strong>Project #2: </strong><br>
                                                 <strong>Leading  lab</strong>:</span> Beliveau Lab<br>
                                                   <span class="aaastandout"><strong>Stakeholder</strong>:</span> Conor Camplisson &lt;concamp [ a t ] uw.edu&gt;<br>
                                                   <span class="aaastandout"><strong>Hackathon  project title</strong>:</span> Genome-wide simulation of <em>in situ </em>hybridization<br>
                                                   <strong class="aaastandout">Desired  deliverable: </strong>A web app that allows users to input  oligonucleotide probe sequence(s) and visualize their predicted binding  genome-wide.<br>
                                                 <strong class="aaastandout">Expected coding experience level: </strong>Intermediate-Advanced. </p>
                                                 <p><strong class="aaastandout">Abstract: </strong><br>
                                                 Several tools exist for designing  oligonucleotide probes for use in fluorescence <em>in situ </em>hybridization (FISH) experiments. However, there is an  unmet need for a tool that can simulate the genome-wide binding profile of  candidate probes under experimental hybridization conditions. Existing tools  vary in the level of sophistication of their approaches to analyzing probe  specificity as well as in the level of coding expertise needed to use the  tools. This project will produce a tool that maximizes both of these  dimensions, leveraging a state of the art specificity analysis pipeline based  on short-read alignment and nearest-neighbor thermodynamics to simulate the  genome-wide binding of candidate probes and visualize the predicted binding  profile under experimental conditions. The app will include a web interface  that can be used without any coding experience, allowing the users to easily  and accurately predict the <em>in situ </em>performance  of their candidate probes. The Beliveau lab has several related web tools  (PaintSHOP, TigerFISH) as well as existing algorithms that will form the  back-end bioinformatics component of the web app. For this project we will need  to create a containerized app that can be deployed to a cloud server, design  and implement the web interface, functionalize the app so that the  bioinformatics algorithms can be integrated into the back-end, and create  infrastructure that allows for the running of analysis jobs in the background  (i.e. using a task queue framework) and exposing front-end parameters so that  users can configure the simulations that run on the back-end.</p>
                                                 <p><strong>&nbsp;</strong></p>
                                                 <p><span class="aaastandout"><strong>Project #3: </strong><br>
                                                 <strong>Leading  lab</strong>:</span> Noble Lab<br>
                                                   <span class="aaastandout"><strong>Stakeholder</strong>:</span> Gang Li (gangliuw [ a t ] uw.edu), Hyeon-Jin Kim (khj3017 [ a t ] uw.edu),  Borislav Hristov (borislav [ a t ] uw.edu)<br>
                                                   <span class="aaastandout"><strong>Hackathon  project title</strong>:</span> Integrating single-cell Hi-C and  single-cell RNA sequencing data<br>
                                                   <strong class="aaastandout">Desired  deliverable: </strong>Benchmarking results comparing multiple  existing software tools for integrating scHiC and scRNA data.<br>
                                                 <strong class="aaastandout">Expected coding experience level: </strong>Intermediate-Advanced. </p>
                                                 <p><span class="aaastandout">Abstract:</span><br>
                                                   Recently, Liu et al. developed HiRES, a  multi-omics sequencing approach to simultaneously profile 3D chromatin contacts  and gene expression in single cells. The HiRES assay thus directly links  chromatin conformation and gene expression profiles within single cells.  Previous studies that aimed to investigate the interplay between chromatin  interactions and transcriptomic profiles necessarily generated unpaired  single-cell RNA-sequencing and chromatin conformation data. For the hackathon,  we will use the newly available HiRES data to benchmark existing software tools  developed for matching cells across modalities. In particular, we evaluate  GLUE, LS-MMDMA (<a href="https://academic.oup.com/bioinformatics/article/39/7/btad420/7221538" target="_blank" class="aaastandout">Meng 2023</a>), Pomona (<a href="https://pubmed.ncbi.nlm.nih.gov/34398192/" target="_blank" class="aaastandout">Cao  2021</a>), SCOT (<a href="https://pubmed.ncbi.nlm.nih.gov/35050714/" target="_blank" class="aaastandout">Demetci 2022</a>), Synmatch (<a href="https://academic.oup.com/bioinformatics/article/38/Supplement_2/ii148/6702005?login=false" target="_blank" class="aaastandout">Hristov 2022</a>), and CMOT (<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02989-8" target="_blank" class="aaastandout">Alatkar 2023</a>). In addition, we will consider  four different techniques for representing the scHi-C contact matrices,  including the contact decay profile, the HiCRep (<a href="https://academic.oup.com/bioinformatics/article/37/18/2996/6133255?login=false" target="_blank" class="aaastandout">Lin 2021</a>) similarity score, a latent Dirichlet  allocation model (<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008173" target="_blank" class="aaastandout">Kim 2020</a>), and the scGAD method (<a href="https://academic.oup.com/bioinformatics/article/38/14/3642/6598798" target="_blank" class="aaastandout">Shen 2022</a>). Our results will compare and  contrast the performance of these tools and Hi-C representation techniques in  the context of integration with scRNA-seq data. <br>
                                                 For the hackathon, we will use co-assay data  to validate integration of scRNA-seq and scHi-C. </p>
                                                 <p><span class="aaastandout"><strong>Project #4</strong><br>
                                                 <strong>Leading  lab</strong>:</span> Nunn lab<br>
                                                   <span class="aaastandout"><strong>Stakeholder</strong>:</span> Brook L. Nunn brookh [ a t ] uw.edu<br>
                                                   <span class="aaastandout"><strong>Hackathon  project title</strong>:</span>   Metagenomic time series portal to interrogate bacterial taxonomic groups  and their functions as they relate to temporal chemical and physical data<br>
                                                 <span class="aaastandout">Expected coding experience level: </span>Beginner(HTML/CSS) - Advanced(experience with  back-end web development) </p>
                                                 <p><span class="aaastandout">Abstract: </span><br>
                                                   The Nunn Lab envisions a transformative  metagenomic time series portal to dissect bacterial taxonomic groups and  functions in relation to temporal chemical and physical data. The portal's  deliverable is a user-friendly website granting access to 133 metagenomes from  ocean water samples, each collected at distinct time points. Currently, the  Joint Genome Institute (JGI) does not link time points together, so you cannot  visualize how a gene is changing through time. <br>
                                                 However, JGI has rich gene annotations. Our  goal is to consolidate all of this data in usable formats. If this data can be  interrogated through a website portal by inquiring how a specific gene or  taxonomic group or keyword located within a gene set is changing through time,  then the data set will be utilized more frequently. The website should allow  users to query using keywords like protein names, KEGG annotations, and PFam annotations,  with results displayed in a comprehensive web page showcasing gene abundance,  taxonomic profiles, and more across the 133 time points. We anticipate the  portal will be utilized in middle school and high school classrooms. In  agreement with JGI when the funding was received we will be publishing a  scientific data report within the year providing all metadata to the public. We  aim to include this portal as part of the publication. </p>
                                                 <p>&nbsp;</p>
                                                 <p><span class="aaastandout"><strong>Project #5</strong><br>
                                                 <strong>Leading  lab</strong>:</span> Stergachis lab<br>
                                                   <span class="aaastandout"><strong>Stakeholder</strong>: </span>Morgan Hamm mhamm [ a t ] uw.edu<br>
                                                   <span class="aaastandout"><strong>Hackathon  project title</strong> : </span> Improving m6A methylation calling from Oxford Nanopore data<br>
                                                 <strong class="aaastandout">Expected coding experience level: </strong>Intermediate-Advanced </p>
                                                 <p><span class="aaastandout">Abstract:</span> <br>
                                                 Fiber-seq is a method to assess chromatin  accessibility using a nonspecific adenine methyltransferase that selectively  labels accessible, but not occluded, adenine bases. Major benefits of this  method over other accessibility techniques is that DNA sequence is preserved  and it is amenable to long-read sequencing. PacBio sequencing is typically used  as the sequencing technology in Fiber-seq assays. Oxford Nanopore sequencing  technology is also capable of calling Adenine methylation, however, methylation  calling is less accurate than with PacBio data, and false positive calls disrupt  downstream analysis steps such as labeling nucleosomes. For this hackathon  project, we will train a model to improve m6A methylation calling for Oxford  Nanopore sequencing data. The model will use the existing methylation calls,  along with quality metrics, and the sequence context of the methylation base  calls to reduce false positive calls. The Stergachis lab has sequenced  methyltransferase treated samples for use in this project. The Stergachis lab  Hackathon project last year performed a similar task, improving methylation  calling for PacBio data, that project led to a currently in review manuscript  for Nature Methods. </p>
<h1>&nbsp;</h1>
                                               <p class="style37">&nbsp;</p>
                                               <p class="style37">&nbsp;</p>
                                               <p class="aaastandout">&nbsp;</p>
											</div>
                                           </article>
                                            

										

									</div>

							</div>
						</div>
					</div>
				</div>

		

		<!-- Scripts -->
			<script src="../assets/js/jquery.min.js"></script>
			<script src="../assets/js/jquery.dropotron.min.js"></script>
			<script src="../assets/js/jquery.slidertron.min.js"></script>
			<script src="../assets/js/skel.min.js"></script>
			<script src="../assets/js/skel-viewport.min.js"></script>
			<script src="../assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="../assets/js/main.js"></script>

	</body>
</html>
