<!DOCTYPE html>

<html lang="en">
    <head>
        
        <link rel="stylesheet" type="text/css" href="templates/slate.css">
            <meta http-equiv="CONTENT-TYPE" content="text/html; charset=UTF-8">
                <meta http-equiv="X-UA-Compatible" content="IE=edge">
                    <meta name="viewport" content="initial-scale=1, width=device-width">
                        <meta content="4DNucleome Hackathon" name="description">
                            <meta content="Projects" name="author">
                                
                                <!-- Global site tag (gtag.js) - Google Analytics -->
                                <script async src="https://www.googletagmanager.com/gtag/js?id=UA-169392828-1"></script>
                                <script>
                                    window.dataLayer = window.dataLayer || [];
                                    function gtag(){dataLayer.push(arguments);}
                                    gtag('js', new Date());
                                    gtag('config', 'UA-169392828-1');
                                </script>
                                    
                                <title>4D Nucleome Hackathon</title>
                                <style>
                                    a:unvisited {color: lightseagreen;}
                                    a:visited {color: lightseagreen;}
                                    a:link {color: lightseagreen;}
                                    a:hover {color: white;}
                                    b {color: lightseagreen;}
                                    body {font-family: 'Book Antiqua', Palatino, serif';font-size: 25px;}
                                </style>
                                <a name="Heading" id="Heading"><div class="jumbotron" style="display: flex; align-items: center;"><a></a>
                                    <img src="4dn_logo_raster.png" style="border-radius: 50%; height: 100px; margin-right: 20px;">
                                    <div>
                                        <h1 style="font-size: 50px; margin: 0;">4D Nucleome Hackathon</h1>
                                        <hr>
                                    </div>
                                </div>
                                <div class="jumbotron">
                                <p style="font-size:18px", align="justify">
                                <a href="#Project 1"><b>Project 1. </b>Finding TF motifs implicated in Hi-C contacts through machine learning</a> &nbsp;&bull;&nbsp <a href="https://docs.google.com/presentation/d/10oDHJKKeddaSJwV6qkwOgyANWwzUsKGm/edit?usp=sharing&ouid=102265118573616376041&rtpof=true&sd=true" target="_blank" class="aaastandout">(Presentation link)</a> <br>
                                <a href="#Project 2"><b>Project 2. </b>Inference of chromatin looping status from live cell imaging data</a> &nbsp;&bull;&nbsp <a href="https://docs.google.com/presentation/d/122gZBw7q_3TRIVxi_QW64fIRSqm4qfD3/edit?usp=sharing&ouid=102265118573616376041&rtpof=true&sd=true" target="_blank" class="aaastandout">(Presentation link)</a> <br>
                                <a href="#Project 3"><b>Project 3. </b>In silico variant prioritization using sequence-based predictive models</a> &nbsp;&bull;&nbsp <a href="https://docs.google.com/presentation/d/1-Ndzp0DAMAwTSP99XGQnxyNVk8EhpOV7/edit?usp=sharing&ouid=102265118573616376041&rtpof=true&sd=true" target="_blank" class="aaastandout">(Presentation link)</a> <br>
                                <a href="#Project 4"><b>Project 4. </b>Integrative analysis of single-cell Hi-C datasets</a> &nbsp;&bull;&nbsp <a href="https://docs.google.com/presentation/d/13mUUgL9HTLzrpjhjl3Yi1zItPdbVuFwx/edit?usp=sharing&ouid=102265118573616376041&rtpof=true&sd=true" target="_blank" class="aaastandout">(Presentation link)</a> <br>
                                <a href="#Project 5"><b>Project 5. </b>Evaluating the effects of Hi-C and TF on cis-regulation of gene expression</a> &nbsp;&bull;&nbsp <a href="https://docs.google.com/presentation/d/1lm0jJOcBX6wAo1ci6TpFYK_o95S2ymeT/edit?usp=sharing&ouid=102265118573616376041&rtpof=true&sd=true" target="_blank" class="aaastandout">(Presentation link)</a> <br>
                                <a href="#Project 6"><b>Project 6. </b>Polymer model benchmarking</a> &nbsp;&bull;&nbsp <a href="https://docs.google.com/presentation/d/1bkaieIvF7fKHzH6a21v-7dZnqo1tBuH1/edit?usp=sharing&ouid=102265118573616376041&rtpof=true&sd=true" target="_blank" class="aaastandout">(Presentation link)</a> <br>
                                <a href="#Project 7"><b>Project 7. </b>Unveiling the role of epigenomic features on RNA splicing throughout neural development using machine learning</a> &nbsp;&bull;&nbsp <a href="https://docs.google.com/presentation/d/11JC6Y5swVnAFxa-NW3GrR2CTtddPWHmI/edit?usp=sharing&ouid=102265118573616376041&rtpof=true&sd=true" target="_blank" class="aaastandout">(Presentation link)</a> <br>
                                <a href="#Project 8"><b>Project 8. </b>SmellEnhancer: Integrating Machine Learning for Enhancer-based Olfactory Receptor Gene Regulation</a> &nbsp;&bull;&nbsp <a href="https://docs.google.com/presentation/d/1DIPI12T4jRyHeX-4G3v7zDigjh-iJj5G/edit?usp=sharing&ouid=102265118573616376041&rtpof=true&sd=true" target="_blank" class="aaastandout">(Presentation link)</a> <br>
                                <a href="#Project 9"><b>Project 9. </b>Building reproducible 3D genome simulations frameworks by populating the polymer model zoo</a> &nbsp;&bull;&nbsp <a href="https://docs.google.com/presentation/d/1zbrmNlXNQFn2H0wZuYMoiHl1WvufkTnp/edit?usp=sharing&ouid=102265118573616376041&rtpof=true&sd=true" target="_blank" class="aaastandout">(Presentation link)</a> <br><br>
                                <a href="index.html" target="_self" class="aaastandout"><b>Back to main</b></a>
                                </p>
                                </div>
                                

    
    </head>
                                        
    <body role="document" style='font="Book Antiqua', Palatino, serif"'>
        
        <a name="Project 1" id="Project 1"><div class="jumbotron" name="Project 1" id="Project 1"><a></a>
                
                <div class="page-header">
                <h2 style="font-size:26px"><b>Project 1. Finding TF motifs implicated in Hi-C contacts through machine learning.</b></h2>
                                                            
                <p style="font-size:17px", align="justify">
                    <b>Leading lab</b>: Noble Lab at University of Washington <br>
                    <b>Stakeholders</b>: Anupama Jha (<a href = "mailto: anupamaj@uw.edu">anupamaj@uw.edu</a>), Xiao Wang (<a href = "mailto: wang3702@uw.edu">wang3702@uw.edu</a>) <br>
                    <b>Desired deliverable</b>:  A benchmark of current Hi-C machine learning models for finding known and novel sequence elements relevant for tissue-specific Hi-C contacts. <br>
                    <b>Expected coding experience level</b>: Intermediate-Advanced. <br>
                    <b>Motivation</b>: Two major classes of machine learning models exist for predicting Hi-C contacts.
The first model type predicts Hi-C contacts from DNA sequence alone, and it is evaluated on
held-out chromosomes in the same tissue. Such models can study the impact of sequence
variations on Hi-C contacts within a tissue. Some examples are Akita, Orca, and DeepC. The
second model type predicts Hi-C contacts by combining DNA sequence with other epigenetic
measurements like ATAC-seq, DNAse-seq and TF-ChIP-seq, and it can predict Hi-C contacts
on held-out chromosomes within a tissue and in new tissues conditional on the availability of the
epigenetic tracks. Some examples are epiphany and Origami. While the prediction performance
of these models has been thoroughly studied, evaluations regarding their interpretation,
especially in the context of capturing sequence elements relevant to tissue-specific gene
regulation, are lacking. <br>
                    <b>Post-Hackathon Summary</b>: Two main classes of machine learning models exist for predicting Hi-C contacts. The first model type predicts Hi-C contacts from DNA sequence alone, and it is evaluated on held-out chromosomes in the same tissue. Such models can study the impact of sequence variations on Hi-C contacts within a tissue. Some examples are Akita, Orca, and DeepC. The second model type predicts Hi-C contacts by combining DNA sequence with other epigenetic measurements like ATAC-seq, DNAse-seq and TF-ChIP-seq, and it can predict Hi-C contacts on held-out chromosomes within a tissue and in new tissues conditional on the availability of the epigenetic tracks. Some examples are Epiphany and C. Origami. While the prediction performance of these models has been thoroughly studied, evaluations regarding their interpretation, especially in the context of capturing sequence elements relevant to tissue-agnostic and tissue-specific gene regulation, are lacking. 

In this project, we systematically evaluated one model from both model classes on their ability to capture tissue-agnostic and tissue-specific gene expression signals, such as transcription factors. We ran the deep learning interpretation framework, Integrated Gradients, to compute attributions for one model from each class, Orca and Epiphany. We then leveraged RNA-seq and ATAC-seq data from GTEx and ENCODE in hundreds of tissues to find tissue-agnostic transcription factors. In parallel, we found tissue-specific transcription factors in three target tissues where we had trained Hi-C models, h1ESC, GM12878 and HFF. Subsequently, we used ATAC-seq data in  h1ESC, GM12878 and HFF to find the accessible regions of the genome.  Then, we searched for motif occurrences of transcription factors in the accessible regions of the held-out chromosomes using FIMO. Finally, we evaluated whether the models found some transcription factors significant for 3D genome structure using attribution scores of motif loci compared to background regions. <br>
<a href="#Heading"><b>Back to top</b></a><br>
</p>
                
        </div></div></a>

        <a name="Project 2" id="Project 2"><div class="jumbotron" name="Project 2" id="Project 2"><a></a>
                
                <div class="page-header">
                <h2 style="font-size:26px"><b>Project 2. Inference of chromatin looping status from live cell imaging data. </b></h2>
                                                            
                <p style="font-size:17px", align="justify">
                    <b>Leading lab</b>: Li lab at University of North Carolina, Chapel Hill (UNC-CH) and Hu lab at Cleveland Clinic <br>
                    <b>Stakeholders</b>: Hongyu Yu (<a href = "mailto: hongyuyu@email.unc.edu">hongyuyu@email.unc.edu</a>) and Shreya Mishra (<a href = "mailto: mishras10@ccf.org">mishras10@ccf.org</a>) <br>
                    <b>Desired deliverable</b>:  (1) Develop a computational pipeline to estimate the frequency and duration of chromatin looping status from live cell imaging data. (2) Characterize the cell-to-cell variability of the kinetics of chromatin looping events. (3) Compare chromatin looping dynamics among different genomic loci. <br>
                    <b>Expected coding experience level</b>: Basic familarity with Python and R. <br>
                    <b>Motivation</b>: The recently developed live cell imaging technology (PMID: 31124784, 30038397, 33310227) provides a powerful tool to study the kinetics of chromatin spatial organization in live cells, facilitating a deep understanding of chromatin folding dynamics and gene regulation. In contrast to rapid development of experimental technologies, tailored computational methods for analyzing live cell imaging data are still lacking. In the project, the goal is to develop a stand-alone, user-friendly computational pipeline to infer the underlying chromatin looping status from live cell imaging data. The software to be developed can be applied to characterize the temporal dynamics of both CTCF-CTCF loops and enhancer-promoter interactions, and has the potential to shed novel insights on transcriptional bursting and gene regulation.<br>
                    <b>Post-Hackathon Summary</b>: <br>
                    <a href="#Heading"><b>Back to top</b></a><br>
                    </p>
                
        </div></div></a>

    <a name="Project 3" id="Project 3"><div class="jumbotron" name="Project 3" id="Project 3"><a></a>
            
            <div class="page-header">
            <h2 style="font-size:26px"><b>Project 3. In silico variant prioritization using sequence-based predictive models. </b></h2>
                                                        
            <p style="font-size:17px", align="justify">
                <b>Leading lab</b>: Pollard Lab at UCSF <br>
                <b>Stakeholders</b>: Katie Gjoni (<a href = "mailto: katie.gjoni@gladstone.ucsf.edu">katie.gjoni@gladstone.ucsf.edu</a>), Shu Zhang (<a href = "mailto: shu.zhang@gladstone.ucsf.edu">shu.zhang@gladstone.ucsf.edu</a>) and Katie Pollard (<a href = "mailto: katherine.pollard@gladstone.ucsf.edu">katherine.pollard@gladstone.ucsf.edu</a>) <br>
                <b>Desired deliverable</b>: Implementing various machine learning models to score variants for disruption to predicted results. <br>
                <b>Expected coding experience level</b>: Strong background in python, and experience working in bash. <br>
                <b>Motivation</b>: Predictive bioinformatics algorithms that take DNA sequences as input are frequently used to
test the effects of genetic variants in high throughput in silico perturbation experiments. There
previously lacked a standard procedure in formatting individual variants into a set of
ready-to-use inputs. However, our lab recently developed SuPreMo, a computational tool to
prepare individual genetic variants into model-ready DNA sequences. In addition, SuPreMo is
directly integrated with Akita, a model that predicts genome folding. This project intends to
extend this work to generate a set of SuPreMo-based tools that comprehensively scores
variants for effects on various chromatin profiles and gene expression. This entails three coding
days of 1) implementing machine learning models, 2) processing their outputs to allow for fair
comparison of outputs, and 3) evaluating statistical methods for comparing outputs from
reference and mutated sequences. These tools would be important for prioritizing putative
pathogenic variants for experimental studies, decoding the grammar of noncoding DNA
sequences, discovering new sequence motifs, designing tissue-specific enhancers, and
uncovering novel roles of sequence elements.<br>
<b>Post-Hackathon Summary</b>: SuPreMo is a pipeline that streamlines in silico mutagenesis (ISM) for sequence-based predictive models and SuPreMo-Akita generated scores for disruption to genome folding. Our team set out to 1) incorporate new models with SuPreMo in addition to Akita, and 2) add additional functionality to SuPreMo. Adding new models to the pipeline allows for faster, easier, and more flexible prediction of variant effects on various features. Our team developed two new versions: SuPreMo-Enformer, which performs ISM on various predicted genomic tracks including gene expression, accessibility, epigenetic modifications, etc. and SuPreMo-ExPecto, which performs ISM with predicted gene expression values for one gene at a time. Now users can take advantage of the flexible SuPreMo pipeline while also being able to predict variant effects on new genomic annotations across different conditions provided by the three models, such as window sizes, cell type, assay type, and variant distance to feature. 

We added functionality that affects both SuPreMo-generated sequences and scores. Users can now input a set of individual-specific SNVs for SuPreMo to generate a personalized reference genome. A use case for this option is when you have variants from both a disease sample and its matched normal. In addition, we created options for sequence mutagenesis for a variant and its surrounding regions. These options include mutating the GC content of the sequence, shuffling the sequence, and mutating transcription factor binding motifs in the sequence. In regards to scoring variants, users can now quantify the disruption of a variant and obtain a p-value when they input a set of control variants. In this case, the query variant score would be compared to the null distribution of scores generated by the control variants. Finally, disruption scores can be weighted based on additional features of the variant. For example, a variant can be prioritized if it disrupts nearby genes, histone marks of active enhancers or promoters, or regions of chromatin accessibility. All of these options increase the possibilities of ISM with SuPreMo.
<br>
<a href="#Heading"><b>Back to top</b></a><br>
            </p>
            
    </div></div></a>

    <a name="Project 4" id="Project 4"><div class="jumbotron" name="Project 4" id="Project 4"><a></a>
            
            <div class="page-header">
            <h2 style="font-size:26px"><b>Project 4. Integrative analysis of single-cell Hi-C datasets. </b></h2>
                                                        
            <p style="font-size:17px", align="justify">
                <b>Leading lab</b>: Jian Ma Lab at CMU <br>
                <b>Stakeholders</b>: Akanksha Sachan (<a href = "mailto: akanksha.11.05.07@gmail.com">akanksha.11.05.07@gmail.com</a>) and Wendy Yang (<a href = "mailto: muyuy@andrew.cmu.edu">muyuy@andrew.cmu.edu</a>) <br>
                <b>Desired deliverable</b>: In this project, we aim to build a workflow for scHi-C data analysis using the
existing software (Higashi and Fast-Higashi) to achieve the following two goals: (1)
Using Higashi/Fast-Higashi to embed single cells and impute scHi-C data; Then use the embedding to call compartments/TADs (2) If time permits, identifying the
single-cell level 3D genome features (sub-compartments) by scGHOST based on the imputed scHi-C contact maps and
study the cell-to-cell variability of those features. <br>
                <b>Expected coding experience level</b>: Python, bash/terminal experience. <br>
                <b>Motivation</b>: Cellular heterogeneity can be observed by embedding their genome architectural
features. This also enables connecting genomic architecture to other epigenetic data
modalities to assess its functional roles comprehensively. Analysing cell-to-cell variability of 3D genome features from sparse scHiC data <br>
<b>Post-Hackathon Summary</b>: Standardizing embedding, imputation, and 3D genomic feature analysis for scHiC datasets is challenging but necessary for mapping the heterogeneity of chromatin architecture across different cell types. To achieve this aim, we created a comprehensive single-cell HiC sequencing data analysis pipeline during the hackathon. We applied the pipeline on two datasets from the human prefrontal cortex (5k cells) and mouse hippocampal formation regions (15k cells) sequenced using sn-m3c-seq. We benchmarked 5 computational methods for generating single-cell embeddings using HiC data, which cluster these cells into their different types/sub-types using quantitative metrics such as ARI, etc for a fair comparison across methods. We evaluated the performance of 2 methods (Higashi and scHiCluster) for imputing the sparsity in the single-cell HiC contact maps and visualizing the intra-chromosomal matrices. We also extended the benchmarking to assess the ability of the methods to perform cell-type clustering using 3D genomic features such as A/B compartment scores computed in single cells. Quantifying such features across different cell types would aid the discovery of cell-type-specific compartments, sub-compartments, and other architectural features playing possible functional roles in expressing genes and replicating DNA.
<br>
<a href="#Heading"><b>Back to top</b></a><br>        
</p>
            
    </div></div></a>

    <a name="Project 5" id="Project 5"><div class="jumbotron" name="Project 5" id="Project 5"><a></a>
            
            <div class="page-header">
            <h2 style="font-size:26px"><b>Project 5. Evaluating the effects of Hi-C and TF on cis-regulation of gene expression. </b></h2>
                                                        
            <p style="font-size:17px", align="justify">
                <b>Leading lab</b>: Christina Leslie Lab at MSKCC <br>
                <b>Stakeholders</b>: Alireza Karbalayghareh (<a href = "mailto: karbalayghareh@gmail.com">karbalayghareh@gmail.com</a>) and Rui Yang (<a href = "mailto: ruy4001@med.cornell.edu">ruy4001@med.cornell.edu</a>) <br>
                <b>Desired deliverable</b>: In this project, we aim to explore the impact of 3D genome structure on gene expression. GraphReg, a deep learning model, leverages 3D interactions along with 1D epigenomic data or genomic DNA sequences to predict gene expression. During the hackathon, participants will have the opportunity to engage in: 
a) Biological application of the model: Participants will apply the GraphReg model to germinal center B cell data. Using feature attributions, they will investigate potential enhancer-promoter interactions or distal regulatory elements influencing gene expression. 
b) Technical benchmarking of the data: Epiphany and ChromaFold are two deep learning models to predict Hi-C contact maps using 1D epigenomic tracks or scATAC-seq matrices. Participants will benchmark the effectiveness of GraphReg predictions by comparing experimental Hi-C data against model-predicted data, and analyze the results.
<br>
                <b>Expected coding experience level</b>: Basic knowledge about Python, bash/terminal experience. <br>
                <b>Motivation</b>: In this project, we hope to explore two directions: a) a biological application of GraphReg on novel dataset; b) Benchmark the application of real experimental data vs. predicted Hi-C data from previously published models.  <br>
                <b>Post-Hackathon Summary</b>: We have investigated the impact of Hi-C and transcription factors (TF) on cis-regulation of gene expression. We included 38 TF ChIP-seq datasets as inputs for GraphReg to examine the influence of TF binding on gene expression. Additionally, we assessed model performance using graphs constructed from real Hi-C, predicted Hi-C, and without a 3D structure. Our findings indicate that incorporating 3D information enhances the model's robustness in predicting in-silico perturbations, as confirmed by TF CRISPR knockout experiments. Notably, while the Epiphany model trained with mean squared error loss produces blurred predictions, it successfully identifies 90% of enhancer-promoter (E-P) interactions from real Hi-C data. Furthermore, GraphReg shows robustness against false-positive interactions, efficiently identifying relevant E-P interactions and accurately predicting gene expression. Moreover, we have calculated SHAP values using GraphReg to explore the role of TF binding in specific gene regulation.<br>
                <a href="#Heading"><b>Back to top</b></a><br>
                </p>
            
    </div></div></a>

        <a name="Project 6" id="Project 6"><div class="jumbotron" name="Project 6" id="Project 6"><a></a>

        <div class="page-header">
        <h2 style="font-size:26px"><b>Project 6. Polymer model benchmarking. </b></h2>
                                                    
        <p style="font-size:17px", align="justify">
            <b>Leading lab</b>: Plewczynski Laboratory from University of Warsaw in Center of New Technologies CeNT, Warsaw, Poland <br>
            <b>Stakeholders</b>: 
            Jedrzej Kubica (<a href = "mailto: j.kubica@cent.uw.edu.pl">j.kubica@cent.uw.edu.pl</a>), Dariusz Plewczynski (<a href = "mailto: d.plewczynski@cent.uw.edu.pl">d.plewczynski@cent.uw.edu.pl</a>) <br> 
            <b>Desired deliverable</b>:  Collection of 3D chromatin models for selected cell lines and experimental assays, protocol for critical assessment of the models, catalog of problems solvable by chromatin modeling, classification of modeling methods. <br>
            <b>Expected coding experience level</b>: Familiar with Python or C. <br>
            <b>Motivation</b>: The aim of the project is the identification of software to be suggested as the state-of-the-art practice in the modeling of spatial chromatin organization. 
            In recent years, various approaches have been taken to predict the structure of chromatin, the majority of which leverages data about epigenomic modifications or imaging data. 
            During the hackathon, we plan to analyze a meta-ensemble of chromatin structure models obtained from state-of-the art software. 
            The rationale behind the project is that different approaches have their advantages, as well as drawbacks. 
            In order for the community of researchers to benefit from the software, it is important to identify opportunities to improve it. 
            The goal of our project is to provide the direction for reaching a high level of confidence in the modeling of chromatin structure.<br>
            <b>Post-Hackathon Summary</b>: <br>
            <a href="#Heading"><b>Back to top</b></a><br>
            </p>

        </div></div></a>

        <a name="Project 7" id="Project 7"><div class="jumbotron" name="Project 7" id="Project 7"><a></a>
        
        <div class="page-header">
        <h2 style="font-size:26px"><b>Project 7. Unveiling the role of epigenomic features on RNA splicing throughout neural development using machine learning. </b></h2>
                                                    
        <p style="font-size:17px", align="justify">
            <b>Leading lab</b>: YIN SHEN Lab from UCSF <br>
            <b>Stakeholders</b>: Jing Wang (<a href = "mailto: Jing.Wang4@ucsf.edu">Jing.Wang4@ucsf.edu</a>), Ian Jones (<a href = "mailto: Ian.Jones3@ucsf.edu">Ian.Jones3@ucsf.edu</a>) <br>
            <b>Desired deliverable</b>:  The deliverable of this study will be a comprehensive analysis outlining correlations between chromatin accessibility profiles obtained from ATAC-seq and PLAC-seq data and the corresponding RNA splicing patterns extracted from RNA-seq data. This will potentially include visualizations, statistical models, and identified key regulatory elements impacting splicing events. <br>
            <b>Expected coding experience level</b>: Basic familarity with Python and R. <br>
            <b>Motivation</b>: This study aims to elucidate the relationship between chromatin accessibility and RNA splicing using multi-omics data obtained from human brain samples. Leveraging RNA-seq, ATAC-seq, and PLAC-seq datasets, our primary objective is to analyze the influence of chromatin accessibility patterns on the splicing landscape across multiple subtypes the human brain.<br>
            <b>Post-Hackathon Summary</b>: Firstly, we need to express our sincere thanks to the supporters and everyone who helped. I believe all members had a very positive and impressive experience during the meeting. Each member was highly active, and with solid support from the hackathon supporters, we had a reliable platform to explore and test everything we wanted. Thanks to the efforts of the hackathon committee staff, we were able to utilize AWS storage, GPU, and CPU resources. Our team applied machine learning to predict RNA splicing events based on epigenetic features. We thoroughly evaluated different models, confirming that random forest and deep learning show the most promise for future analysis. We plan to continue our analysis after the hackathon, and we believe our findings could contribute to impactful research in the near future. Lastly, but certainly not least, thank you to everyone who supported the hackathon.<br>
            <a href="#Heading"><b>Back to top</b></a><br>
            </p>
        
    </div></div></a>

    <a name="Project 8" id="Project 8"><div class="jumbotron" name="Project 8" id="Project 8"><a></a>
        
        <div class="page-header">
        <h2 style="font-size:26px"><b>Project 8. SmellEnhancer: Integrating Machine Learning for Enhancer-based Olfactory Receptor Gene Regulation. </b></h2>
                                                    
        <p style="font-size:17px", align="justify">
            <b>Leading lab</b>: Lomvardas Lab at Columbia University <br>
            <b>Stakeholders</b>: Miao Wang (<a href = "mailto: mw3777@columbia.edu">mw3777@columbia.edu</a>),  Isabella Pirozzolo (<a href = "mailto: idp2121@cumc.columbia.edu">idp2121@cumc.columbia.edu</a>). <br>
            <b>Desired deliverable</b>:  Generate models to investigate the cooperative regulation of enhancers on gene expression in cell development. <br>
            <b>Expected coding experience level</b>: Basic knowledge about Python, bash/terminal experience. <br>
            <b>Motivation</b>: Enhancer-promoter interactions drive cell differentiation and cell fate determination. However, how cell type-specific enhancers cooperate during these processes remains unknown. The olfactory system detects and identifies odor signals with the “one receptor per neuron” rule where each mature olfactory sensory neuron (mOSN) expresses only one out of more than ~2000 olfactory gene (OR) alleles. OR gene activation is accomplished by forming chromatin interactions between OR genes and OR-specific enhancer elements named Greek Islands (GIs).  At least 63 GIs were distributed in the intergenic regions in the mice.  Therefore, olfactory receptor genes provide an ideal system to study how enhancers cooperate to regulate gene transcription transition in cell fate decisions. <br>
            <b>Post-Hackathon Summary</b>: Olfactory sensory neurons express one olfactory receptor (OR) out of a thousand choices. The chosen OR outcompetes multiple others that are expressed in developing cells. We aimed to develop a model to predict the chosen OR from a set of regulatory elements. Using a multiome dataset that combined RNAseq with ATACseq in the same single cells, we identified the chosen olfactory receptor in each cell, extracted the ATACseq peaks over regulatory sequences of interest, and used these cleaned data files to build several models. We built and optimized linear regression, random forest, and decision tree models. These were largely unable to predict OR choice or expression level. They were limited both by data sparsity and the lack of information on additional elements important for choice. Finally, we built a stacked model designed to predict one of the top five chosen ORs. It had a higher accuracy, suggesting that chromatin accessibility may be partially deterministic in OR choice. We will continue to build on this model and deepen datasets that could unpack the rules governing this process.<br>
            <a href="#Heading"><b>Back to top</b></a><br>
            </p>
        
    </div></div></a>

    <a name="Project 9" id="Project 9"><div class="jumbotron" name="Project 9" id="Project 9"><a></a>
        
        <div class="page-header">
        <h2 style="font-size:26px"><b>Project 9. Building reproducible 3D genome simulations frameworks by populating the polymer model zoo. </b></h2>
                                                    
        <p style="font-size:17px", align="justify">
            <b>Leading lab</b>: Fudenberg lab at USC <br>
            <b>Stakeholders</b>: Sasha Galitsyna (<a href = "mailto: galitsyn@mit.edu">galitsyn@mit.edu</a>), Max Tortora (<a href = "mailto: tortora@usc.edu">tortora@usc.edu</a>), Geoff Fudenberg (<a href = "mailto: fudenber@usc.edu">fudenber@usc.edu</a>)<br>
            <b>Desired deliverable</b>:  Initial commits for the 3D genome polymer model zoo and a draft design document to unify polymer model specifications. <br>
            <b>Expected coding experience level</b>: Python background. Prior experience with: biophysics, file formats, and open source software development would be helpful. <br>
            <b>Motivation</b>: Despite progress in reproducible analysis of experimental data, software for the specification and analysis of biophysical simulations of 3D genomes has lagged. Existing simulation engines, often adapted from protein folding applications (e.g., HooMD, OpenMM, LAMMPS), lack modularity and tend toward complex, unsustainably coded frameworks. The Open Chromosome Collective (Open2C) aims to remedy this by developing an open-source polymer modeling framework for 3D genome simulation, polychrom. Exploration and implementation of published polymer models into a unified codebase are needed to develop a robust and flexible high-level API. Hackathon team members will build polymer models of 3D genome organization across cell states and organisms, populate a polymer model zoo, and understand the design principles for a robust polymer simulation framework.<br>
            <b>Post-Hackathon Summary</b>: 3D genome polymer simulations hold significant importance in understanding chromatin dynamics and Hi-C data interpretation. Simulation software typically presents a steep learning curve, requiring knowledge of physics, dealing with complex programming interfaces, and enduring extensive run times. To remedy this, the main aim of our project was to differentially assess — and potentially improve — the reliability, performance and API friendliness of various polymer simulations frameworks for coarse-grained chromatin simulations. By benchmarking various molecular dynamics schemes, we were able to show that the hydrodynamics-based dissipative-particle dynamics (DPD) method significantly outperformed the more commonly-used Langevin integrator for simulation of large (>100,000 particles) polymeric regions, without loss of accuracy. We further highlighted how this DPD backend may be tractably applied in the context of chromatin biology by devising a novel pipeline based on polychrom-hoomd (https://github.com/open2c/polychrom-hoomd) for quantitative simulations of single-cell HiC data. We finally deployed a new interactive API based on WandB (https://wandb.ai) for the live-tracking of simulation results and performance. The project also involved the development of streamlined tools to facilitate the integration of experimental data such as compartment profiles or single-cell HiC constraints into the simulation workflow, as well as the implementation of a robust analysis pipeline to be able to process both experimental and simulation data within a unified software ecosystem. The codes generated during the hackathon may be found here: https://github.com/4dn-hack-team9-2024.
<br>
<a href="#Heading"><b>Back to top</b></a><br>
            </p>
        
    </div></div></a>

    </body>

</html>
