<!DOCTYPE html>

<html lang="en">
    <head>
        
        <link rel="stylesheet" type="text/css" href="templates/slate.css">
            <meta http-equiv="CONTENT-TYPE" content="text/html; charset=UTF-8">
                <meta http-equiv="X-UA-Compatible" content="IE=edge">
                    <meta name="viewport" content="initial-scale=1, width=device-width">
                        <meta content="4DNucleome Hackathon" name="description">
                            <meta content="Projects" name="author">
                                
                                <!-- Global site tag (gtag.js) - Google Analytics -->
                                <script async src="https://www.googletagmanager.com/gtag/js?id=UA-169392828-1"></script>
                                <script>
                                    window.dataLayer = window.dataLayer || [];
                                    function gtag(){dataLayer.push(arguments);}
                                    gtag('js', new Date());
                                    gtag('config', 'UA-169392828-1');
                                </script>
                                    
                                <title>4D Nucleome Hackathon</title>
                                <style>
                                    a:unvisited {color: lightseagreen;}
                                    a:visited {color: lightseagreen;}
                                    a:link {color: lightseagreen;}
                                    a:hover {color: white;}
                                    b {color: lightseagreen;}
                                    body {font-family: 'Book Antiqua', Palatino, serif';font-size: 25px;}
                                </style>
                                <div class="jumbotron" style="display: flex; align-items: center;">
                                    <img src="4dn_logo_raster.png" style="border-radius: 50%; height: 100px; margin-right: 20px;">
                                    <div>
                                        <h1 style="font-size: 50px; margin: 0;">4D Nucleome Hackathon</h1>
                                        <hr>
                                    </div>
                                </div>
                                <div class="jumbotron">
                                <p style="font-size:18px", align="justify">
                                <a href="#Project 1">Project 1</a> &nbsp;&bull;&nbsp
                                <a href="#Project 2">Project 2</a> &nbsp;&bull;&nbsp
                                <a href="#Project 3">Project 3</a> &nbsp;&bull;&nbsp
                                <a href="#Project 4">Project 4</a> &nbsp;&bull;&nbsp
                                <a href="#Project 5">Project 5</a> &nbsp;&bull;&nbsp
                                <a href="#Project 6">Project 6</a> &nbsp;&bull;&nbsp
                                <a href="#Project 7">Project 7</a> &nbsp;&bull;&nbsp
                                <a href="index.html" target="_blank" class="aaastandout">Back to main</a>
                                </p>
                                </div>
                                

    
    </head>
                                        
    <body role="document" style='font="Book Antiqua', Palatino, serif"'>
        
        <a name="Project 1" id="Project 1"><div class="jumbotron" name="Project 1" id="Project 1"><a></a>
                
                <div class="page-header">
                <h2 style="font-size:26px"><b>Project 1. Find sequence elements implicated in tissue-specific Hi-C contacts through machine learning.</b></h2>
                                                            
                <p style="font-size:20px", align="justify">
                    <b>Leading lab</b>: Noble Lab <br>
                    <b>Stakeholders</b>: Anupama Jha (anupamaj [ a t ] uw.edu) and Gang Li (gangliuw [ a t ] uw.edu) <br>
                    <b>Desired deliverable</b>:  A benchmark of current Hi-C machine learning models for finding known and novel sequence elements relevant for tissue-specific Hi-C contacts. <br>
                    <b>Expected coding experience level</b>: Intermediate-Advanced. <br>
                    <b>Motivation</b>: Two major classes of machine learning models exist for predicting Hi-C contacts.
The first model type predicts Hi-C contacts from DNA sequence alone, and it is evaluated on
held-out chromosomes in the same tissue. Such models can study the impact of sequence
variations on Hi-C contacts within a tissue. Some examples are Akita, Orca, and DeepC. The
second model type predicts Hi-C contacts by combining DNA sequence with other epigenetic
measurements like ATAC-seq, DNAse-seq and TF-ChIP-seq, and it can predict Hi-C contacts
on held-out chromosomes within a tissue and in new tissues conditional on the availability of the
epigenetic tracks. Some examples are epiphany and Origami. While the prediction performance
of these models has been thoroughly studied, evaluations regarding their interpretation,
especially in the context of capturing sequence elements relevant to tissue-specific gene
regulation, are lacking. <br>
                </p>
                
        </div></div></a>

        <a name="Project 2" id="Project 2"><div class="jumbotron" name="Project 2" id="Project 2"><a></a>
                
                <div class="page-header">
                <h2 style="font-size:26px"><b>Project 2. Inference of chromatin looping status from live cell imaging data. </b></h2>
                                                            
                <p style="font-size:20px", align="justify">
                    <b>Leading lab</b>: Li lab at University of North Carolina, Chapel Hill (UNC-CH) and Hu lab at Cleveland Clinic <br>
                    <b>Stakeholders</b>: Hongyu Yu (hongyuyu@email.unc.edu) and Shreya Mishra (mishras10@ccf.org) <br>
                    <b>Desired deliverable</b>:  (1) Develop a computational pipeline to estimate the frequency and duration of chromatin looping status from live cell imaging data. (2) Characterize the cell-to-cell variability of the kinetics of chromatin looping events. (3) Compare chromatin looping dynamics among different genomic loci. <br>
                    <b>Expected coding experience level</b>: Basic familarity with Python and R. <br>
                    <b>Motivation</b>: The recently developed live cell imaging technology (PMID: 31124784, 30038397, 33310227) provides a powerful tool to study the kinetics of chromatin spatial organization in live cells, facilitating a deep understanding of chromatin folding dynamics and gene regulation. In contrast to rapid development of experimental technologies, tailored computational methods for analyzing live cell imaging data are still lacking. In the project, the goal is to develop a stand-alone, user-friendly computational pipeline to infer the underlying chromatin looping status from live cell imaging data. The software to be developed can be applied to characterize the temporal dynamics of both CTCF-CTCF loops and enhancer-promoter interactions, and has the potential to shed novel insights on transcriptional bursting and gene regulation.<br>
                </p>
                
        </div></div></a>

    </body>

</html>
